interface/patient_file/report/custom_report.php

```diff
@@ -222,751 +222,755 @@ function zip_content($source, $destination, $content = '', $create = true)
-                     $form_dir_arr[] = add_escape_custom($res_search[1]);
-        }
-    }
+<body style="<?php echo $style; ?>">
+    <div class="container">
 
-    //echo json_encode(json_encode($array_key_id));
-    if (sizeof($form_id_arr) > 0) {
-        $query = "SELECT DISTINCT(form_name),formdir FROM forms WHERE form_id IN ( '" . implode("','", $form_id_arr) . "') AND formdir IN ( '" . implode("','", $form_dir_arr) . "')";
-        $arr = sqlStatement($query);
-        echo "<select multiple size='4' style='width:300px;' id='forms_to_search' onchange='clear_last_visit();remove_mark_all();find_all();' >";
-        while ($res_forms_ids = sqlFetchArray($arr)) {
-            echo "<option value='" . attr($res_forms_ids['formdir']) . "' selected>" . text($res_forms_ids['form_name']) . "</option>";
-        }
+        <div id="report_custom w-100">  <!-- large outer DIV -->
+            <?php
+            if (sizeof($_GET) > 0) {
+                $ar = $_GET;
+            } else {
+                $ar = $_POST;
+            }
 
-        echo "</select>";
-    }
-    ?>
-      </td>
-      <td style="padding-left: 10px;; width: 30%;">
-    <span id ='alert_msg' style='color: red;'></span>
-      </td>
-    </tr>
-  </table>
-    </div>
-    <?php
-} // end not printable ?>
+            if ($printable) {
+                /*******************************************************************
+                 $titleres = getPatientData($pid, "fname,lname,providerID");
+                $sql = "SELECT * FROM facility ORDER BY billing_location DESC LIMIT 1";
+                *******************************************************************/
+                $titleres = getPatientData($pid, "fname,lname,providerID,DATE_FORMAT(DOB,'%m/%d/%Y') as DOB_TS");
+                $facility = null;
+                if ($_SESSION['pc_facility']) {
+                    $facility = $facilityService->getById($_SESSION['pc_facility']);
+                } else {
+                    $facility = $facilityService->getPrimaryBillingLocation();
+                }
 
-<?php
+                /******************************************************************/
+                // Setup Headers and Footers for mPDF only Download
+                // in HTML view it's just one line at the top of page 1
+                echo '<page_header style="text-align:right;" class="custom-tag"> ' . xlt("PATIENT") . ':' . text($titleres['lname']) . ', ' . text($titleres['fname']) . ' - ' . text($titleres['DOB_TS']) . '</page_header>    ';
+                echo '<page_footer style="text-align:right;" class="custom-tag">' . xlt('Generated on') . ' ' . text(oeFormatShortDate()) . ' - ' . text($facility['name']) . ' ' . text($facility['phone']) . '</page_footer>';
+
+                // Use logo if it exists as 'practice_logo.gif' in the site dir
+                // old code used the global custom dir which is no longer a valid
```

Comment:
Hi @tywrenn  and @bradymiller 

Such as this `page_header` tag, is it a valid HTML tag or have a particular feature?

Reply:
It isn't a valid HTML tag and can be replaced with a div tag.

Reply:
umm please be careful with generalities such as if a tag doesn't look to be html. There are times when a developer will create custom tags to parse themselves. However, in this case I believe this tag is being echoed for another parser. i.e mPDF. Not going to go look at code but off top of my head, this is valid.